src/main/no/nav/pto/veilarbfilter/service/EnhetFilterServiceImpl.kt

```diff
@@ -6,49 +6,56 @@ import no.nav.pto.veilarbfilter.db.Filter
 import no.nav.pto.veilarbfilter.model.EnhetensLagredeFilterModel
 import no.nav.pto.veilarbfilter.model.FilterModel
 import no.nav.pto.veilarbfilter.model.NyttFilterModel
+import no.nav.pto.veilarbfilter.model.SortOrder
 import org.jetbrains.exposed.sql.*
 import java.time.LocalDateTime
 
 class EnhetFilterServiceImpl() : FilterService {
 
     override suspend fun hentFilter(filterId: Int): EnhetensLagredeFilterModel? = dbQuery {
         (Filter innerJoin EnhetensLagredeFilter).slice(
-            Filter.filterId,
-            Filter.filterNavn,
-            Filter.valgteFilter,
-            EnhetensLagredeFilter.enhetId
+                Filter.filterId,
+                Filter.filterNavn,
+                Filter.valgteFilter,
+                EnhetensLagredeFilter.enhetId,
+                EnhetensLagredeFilter.sortOrder
         ).select { (Filter.filterId.eq(filterId)) }
-            .mapNotNull { tilEnhetFilterModel(it) }
-            .singleOrNull()
+                .mapNotNull { tilEnhetFilterModel(it) }
+                .singleOrNull()
     }
 
     override suspend fun finnFilterForFilterBruker(enhetId: String): List<EnhetensLagredeFilterModel> = dbQuery {
         (Filter innerJoin EnhetensLagredeFilter).slice(
-            Filter.filterId,
-            Filter.filterNavn,
-            Filter.valgteFilter,
-            EnhetensLagredeFilter.enhetId
+                Filter.filterId,
+                Filter.filterNavn,
+                Filter.valgteFilter,
+                EnhetensLagredeFilter.enhetId,
+                EnhetensLagredeFilter.sortOrder
         ).select { (EnhetensLagredeFilter.enhetId.eq(enhetId)) }
-            .mapNotNull { tilEnhetFilterModel(it) }
+                .mapNotNull { tilEnhetFilterModel(it) }
     }
 
     override suspend fun slettFilter(filterId: Int, enhetId: String): Int = dbQuery {
         val removedRows =
-            EnhetensLagredeFilter.deleteWhere { (EnhetensLagredeFilter.filterId eq filterId) and (EnhetensLagredeFilter.enhetId eq enhetId) }
+                EnhetensLagredeFilter.deleteWhere { (EnhetensLagredeFilter.filterId eq filterId) and (EnhetensLagredeFilter.enhetId eq enhetId) }
         if (removedRows > 0) {
             Filter.deleteWhere { (Filter.filterId eq filterId) }
         } else {
             0
         }
     }
 
+    override suspend fun lagreSortering(filterBrukerId: String, sortOrder: List<SortOrder>): Boolean {
+        TODO("Not yet implemented")
```

Comment:
Good question, I added _lagreSortering_ in interface _FilterService_, since I thought we will maybe implement similar sortring for veiledergruppe or enhet filter. Although I think we can remove enhet filter, since we are using mine filter for such purposes.